test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -683,12 +683,14 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 
 			By("waiting for some of the pods to be running")
 			dpKey := client.ObjectKeyFromObject(dp)
+			time.Sleep(2 * time.Minute) //DELETE_ME
```

Comment:
Given that we are already waiting 5 minutes for the replicas to be created later, this delay seems unnecessary here. Also, I see a comment DELETE_ME here. Was this was added for testing purposes and the intention was to remove it or there is any other reason behind it that I am missing here?

Reply:
need to be removed, it was just for debugging purposes, thanks for pointing this, will address it as well